src/components/pacbio/PacbioPoolTubeItem.vue

```diff
@@ -6,6 +6,9 @@
     @dragstart="drag(barcode, $event)"
     @click="expanded = !expanded"
   >
+    <div class="flex justify-end content-end items-end">
```

Comment:
I dont think `content-end` or `items-end` do anything here positionally so id remove them unless there is a specific reason.